### PR TITLE
refactored test/cmd/help to use wrapper functions

### PR DESCRIPTION
**Pull request by stevekuznetsov**
/cc @deads2k 

@smarterclayton @liggitt PTAL

### CODE REVIEW

File: test/cmd/help.sh [hunk 1]
```diff
@@ -6,111 +6,112 @@ set -o pipefail
...
-[ "$(openshift cli help patch 2>&1 | grep 'openshift')" ]
+os::cmd::expect_success_and_text 'oc help get' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift help cli get' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift help kubectl get' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift help start' 'Start an all-in-one server'
+os::cmd::expect_success_and_text 'openshift help start master' 'Start a master'
+os::cmd::expect_success_and_text 'openshift help start node' 'Start a node'
+os::cmd::expect_failure_and_text 'openshift cli help update' 'openshift'
+os::cmd::expect_failure_and_text 'openshift cli help replace' 'openshift'
+os::cmd::expect_failure_and_text 'openshift cli help patch' 'openshift'
```
Comment (stevekuznetsov) [inline test/cmd/help.sh]: The three expressions here:
```
openshift cli help update
openshift cli help replace
openshift cli help patch
```

Are not valid commands - their error output is:
```
error: unknown command "help replace"
See 'openshift cli -h' for help and examples.
```

What is their purpose?

Comment (deads2k) [inline test/cmd/help.sh]: > What is the purpose of these tests?

Those tests are failing.  `openshift cli help update` should be the same as `oc help update`

Comment (stevekuznetsov) [inline test/cmd/help.sh]: Let's call this three wins for `os::cmd`

File: test/cmd/help.sh [hunk 2]
```diff
@@ -6,111 +6,113 @@ set -o pipefail
...
-[ "$(openshift cli help replace 2>&1 | grep 'openshift')" ]
-[ "$(openshift cli help patch 2>&1 | grep 'openshift')" ]
+os::cmd::expect_success_and_text 'oc help get' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift help cli get' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift help kubectl get' 'Display one or many resources'
+os::cmd::expect_success_and_text 'openshift help start' 'Start an all-in-one server'
+os::cmd::expect_success_and_text 'openshift help start master' 'Start a master'
+os::cmd::expect_success_and_text 'openshift help start node' 'Start a node'
+# TODO: fix these tests
+# os::cmd::expect_success_and_text 'openshift cli help update' 'Update a resource'
```
Comment (deads2k) [inline test/cmd/help.sh]: > Those tests are failing. openshift cli help update should be the same as oc help update

Link me to an issue please.

Comment (stevekuznetsov) [inline test/cmd/help.sh]: Two separate issues: https://github.com/openshift/origin/issues/6042 https://github.com/openshift/origin/issues/6043

**Comment by deads2k** [comment]
lgtm [merge]

**Comment by stevekuznetsov** [comment]
flake on e2e: 
```
>> /data/src/github.com/openshift/origin/assets/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver/promise.js:1761
>>       throw error;
>>             ^
>> Error: Timed out waiting for the WebDriver server at http://172.18.4.193:47669/wd/hub
>>     at Error (<anonymous>)
>>     at onResponse (/data/src/github.com/openshift/origin/assets/node_modules/protractor/node_modules/selenium-webdriver/http/util.js:87:11)
>>     at /data/src/github.com/openshift/origin/assets/node_modules/protractor/node_modules/selenium-webdriver/http/util.js:42:21
>>     at /data/src/github.com/openshift/origin/assets/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver/http/http.js:96:5
>>     at ClientRequest.<anonymous> (/data/src/github.com/openshift/origin/assets/node_modules/protractor/node_modules/selenium-webdriver/http/index.js:145:7)
>>     at ClientRequest.emit (events.js:95:17)
>>     at Socket.socketErrorListener (http.js:1552:9)
>>     at Socket.emit (events.js:95:17)
>>     at net.js:441:14
>>     at process._tickCallback (node.js:442:13)
Fatal error: protractor exited with code: 8


Execution Time (2015-11-24 14:55:33 UTC)
concurrent:server   5.5s  ▇▇▇▇▇▇▇ 15%
protractor:chrome  30.4s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 84%
Total 36.2s


[FAIL] !!!!! Test Failed !!!!
```

**Comment by deads2k** [comment]
> flake on e2e:

Find an issue or open one and tag @jwforres

**Comment by stevekuznetsov** [comment]
Issue already created https://github.com/openshift/origin/issues/6053

**Pull request merged**